packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -389,6 +415,7 @@ export function ArticleEditor({id}: ArticleEditorProps) {
           }}
         />
       </Modal>
+      <div style={{textAlign: 'right', padding: '20px'}}> {`${t("articleEditor.panels.totalCharCount")} ${totalCharCount}`} </div>
```

Comment:
Don't show the character count here. Show it in the metadata panel before the pre title.